code/game/jobs/access_datum.dm

```diff
@@ -454,7 +454,7 @@
 /datum/access/merchant
 	id = access_merchant
 	desc = "Merchant Access"
-	access_type = ACCESS_TYPE_NONE
+	access_type = ACCESS_TYPE_CENTCOM

```

Comment:
Is this really a good idea? It'd give them access to CentComm at round end, and potentially allow them to do shenanigans with ERT and Admin shuttles.